src/test/routes/logout.ts

```diff
@@ -21,6 +21,7 @@ describe('Logout receiver', () => {
   describe('on GET', () => {
     it('should remove session cookie', async () => {
       idamServiceMock.resolveRetrieveUserFor('1', 'citizen')
+      idamServiceMock.resolveInvalidateCode('ABC')
```

Comment:
Add missing test case for the following scenarios:
- session invalidation failed (500 error from the API - cookie still deleted)
- session does not exist (no session cookie - no API call, no cookie deleted)

Reply:
done